kubectl-pg/cmd/addUser.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright © 2019 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+	PostgresqlLister "github.com/zalando/postgres-operator/pkg/generated/clientset/versioned/typed/acid.zalan.do/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"strings"
+)
+
+// addUserCmd represents the addUser command
+var addUserCmd = &cobra.Command{
+	Use:   "add-user",
+	Short: "Add user cmd is to add user to the postgres cluster and it's roles",
+	Long:  `Add user cmd is to add user to the postgres cluster you can add privileges as well with -f flag.`,
```

Comment:
I would cut out "... and it's roles". It's wrong. It could be "with given privileges"
`-p` flag? :)

Reply:
Okay, will change this as well as required.